tests/Php73/Php73Test.php

```diff
@@ -26,6 +26,8 @@ public function testIsCountable()
         $this->assertTrue(is_countable(array(1, 2, '3')));
         $this->assertTrue(is_countable(new \ArrayIterator(array('foo', 'bar', 'baz'))));
         $this->assertTrue(is_countable(new \ArrayIterator()));
+        $this->assertTrue(is_countable(new \SimpleXMLElement('<foo><bar/><bar/><bar/></foo>')));
+        $this->assertTrue(is_countable(\ResourceBundle::create('en', __DIR__.'/fixtures')));
```

Comment:
Don't we need `class_exists()` calls for these two to make sure `intl` and `xml` extensions are loaded.

Reply:
In Travis the extensions are loaded so the classes should always exist. In the actual polyfill, the check would just return `false` if the extension isn't loaded and the class doesn't exist